.travis.yml

```diff
@@ -23,6 +23,8 @@ script:
   - coverage run --source foca -m pytest
   - coveralls
   - cd examples/petstore && docker-compose up --build -d
+  - sleep 10
+  - pytest ../../integration_tests.py
```

Comment:
Yes, they can be added directly under the Travis file but for better readability and easy debugging, I thought to use PyTest. Also, I thought If in case more features are added to the foca in the future, more integration tests would be added making the Travis File difficult to read and understand. Anyways, I will update the pr with curl tests.